exampleCourse/questions/demo/randomFakeData/server.py

```diff
@@ -23,7 +28,7 @@ def generate(data):
     for alias, birthday in zip(
         ["youngest", "employee1", "employee2", "employee3"], birthdays
     ):
-        name = fake.name()
+        name = names.pop()
```

Comment:
```suggestion
        name = fake.unique.name()
```
I believe this is the only change that is needed: https://faker.readthedocs.io/en/master/index.html?highlight=unique#unique-values

Reply:
Neat! I should have figured that they'd have some clever solution to this. Changed in 041af21.